datastructs/counters/lossy_counter/lossycon.c

```diff
@@ -0,0 +1,412 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netdb.h>
+#include <unistd.h>
+#include <inttypes.h>
+#include <string.h>
+#include <arpa/inet.h>
+/****************************************
+        Author: Tim Wood
```

Comment:
remove my name and the link to beej.us since this isn't using any of my networking code.